tests/plugins/contextmenu/manual/hide.html

```diff
@@ -0,0 +1,24 @@
+<style>
+	.marker {
+		width: 50px;
+		height: 50px;
+		background: blue;
+		margin: 5px;
+	}
+</style>
+<textarea id="editor"></textarea>
+<div class="marker"></div>
+<script>
+	if ( bender.tools.env.mobile ) {
```

Comment:
Wouldn't it be possible to make this test work also on mobile devices by using `menubutton` plugin instead of `contextmenu` one?

It would also fix the issue with the fact that clicking the right button inside editor (especially empty one) can also show the native menu instead of desired one

Reply:
I'll just add button to open context menu on mobiles.